identity/manager.go

```diff
@@ -0,0 +1,93 @@
+package identity
+
+import (
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"fmt"
+	"github.com/mysterium/node/service_discovery/dto"
+	"strings"
+)
+
+const PASSPHRASE = ""
+
+type IdentityManager struct {
+	keystoreManager *keystore.KeyStore
+}
+
+func NewIdentityManager(keydir string) *IdentityManager {
+	return &IdentityManager{
+		keystoreManager: keystore.NewKeyStore(keydir, keystore.StandardScryptN, keystore.StandardScryptP),
+	}
+}
+
+func accountToIdentity(account accounts.Account) dto.Identity {
+	return dto.Identity(account.Address.Hex())
+}
+
+func identityToAccount(identityString string) accounts.Account {
+	return accounts.Account{
+		Address: common.HexToAddress(identityString),
+	}
+}
+
+func (idm *IdentityManager) CreateNewIdentity() (dto.Identity, error) {
+	account, err := idm.keystoreManager.NewAccount(PASSPHRASE)
+	if err != nil {
+		return "", err
+	}
+
+	return accountToIdentity(account), nil
+}
+
+func (idm *IdentityManager) GetIdentities() []dto.Identity {
+	var ids []dto.Identity
+	for _, account := range idm.keystoreManager.Accounts() {
+		ids = append(ids, accountToIdentity(account))
+	}
+
+	return ids
+}
+
+func (idm *IdentityManager) GetIdentity(identityString string) *dto.Identity {
+	identityString = strings.ToLower(identityString)
+	for _, id := range idm.GetIdentities() {
+		if strings.ToLower(string(id)) == identityString {
```

Comment:
1. make `accountToIdentity` to lowercase, so You dont need to lower in many places
2. do we need to change format at all, we can break consistency with Ethereum

Reply:
we don't need to do `.ToLower()` here at all.
I'd rather do `id.ToString()` to compare here.